malariagen_data/anoph/sample_metadata.py

```diff
@@ -164,37 +164,21 @@ def _cohorts_metadata_paths(self, *, sample_sets: List[str]) -> Dict[str, str]:
 
     @property
     def _cohorts_metadata_columns(self):
-        # Handle changes to columns used in different analyses.
         cols = None
         if self._cohorts_analysis:
-            if self._cohorts_analysis < "20230223":
-                cols = (
-                    "country_iso",
-                    "admin1_name",
-                    "admin1_iso",
-                    "admin2_name",
-                    "taxon",
-                    "cohort_admin1_year",
-                    "cohort_admin1_month",
-                    "cohort_admin2_year",
-                    "cohort_admin2_month",
-                )
-            # We assume that cohorts analyses from "20230223" onwards always include quarter
-            # columns.
-            else:
-                cols = (
-                    "country_iso",
-                    "admin1_name",
-                    "admin1_iso",
-                    "admin2_name",
-                    "taxon",
-                    "cohort_admin1_year",
-                    "cohort_admin1_month",
-                    "cohort_admin1_quarter",
-                    "cohort_admin2_year",
-                    "cohort_admin2_month",
-                    "cohort_admin2_quarter",
-                )
```

Comment:
I attempted to simplify this as we only have cohort (set) data for cohorts versions post 20230223. So, any version before this will not have data available. We could, of course, retrospectively generate the data for all cohorts we have but not sure how useful would this be. 

Agree this shouldn't mean that we should stop supporting the pre 20230223 cohorts! Will have to downgrade the analysis version in one of the fixture to ensure this gets tested, as I think the coverage is failing because I have updated all cohorts analysis across the API to post versions.